### PR TITLE
Add charset encoding for text/* content-type serializations

### PR DESCRIPTION
**Pull request by acoburn**
Resolves: https://jira.duraspace.org/browse/FCREPO-1750

### CODE REVIEW

**Comment by awoods** [comment]
Is there a reason [FedoraVersions](https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraVersions.java#L141) is not included in this PR?

**Comment by acoburn** [comment]
@awoods: I thought it had been, I will add that in an additional commit

**Comment by awoods** [comment]
Please rebase on master.

**Comment by acoburn** [comment]
@awoods: rebased.

File: fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/responses/RdfStreamProvider.java [hunk 1]
```diff
@@ -52,6 +52,11 @@
 
     private static final Logger LOGGER = getLogger(RdfStreamProvider.class);
 
+    private static boolean isTextHtml(final MediaType mediaType) {
```
Comment (awoods) [inline fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/responses/RdfStreamProvider.java]: This `private` method is not referenced and can be removed.

Comment (acoburn) [inline fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/responses/RdfStreamProvider.java]: Yes it is: https://github.com/fcrepo4/fcrepo4/pull/1112/files#diff-479f0eb6453708a90e8c7cc92f0657a6R222

Comment (awoods) [inline fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/responses/RdfStreamProvider.java]: There are two instances of `isTextHtml`. One in:
StreamingBaseHtmlProvider : https://github.com/fcrepo4/fcrepo4/pull/1112/files#diff-479f0eb6453708a90e8c7cc92f0657a6R238

and another, here, in:
RdfStreamProvider.

The one in RdfStreamProvider is not used.

Comment (acoburn) [inline fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/responses/RdfStreamProvider.java]: Very true. You were looking closer than I was. :-)

**Pull request merged**